datahub-graphql-core/src/main/resources/timeline.graphql

```diff
@@ -0,0 +1,189 @@
+extend type Query {
+    """
+    Get list of schema blame
```

Comment:
This doesn't say much about what schema blame is. just a comment

Reply:
Updated with something more description